playbooks/archive_artifacts.yml

```diff
@@ -0,0 +1,77 @@
+- hosts: "{{ hosts }}"
+  tasks:
+    - name: Create temporary directory to store artifacts
+      file:
+        path: "{{ artifacts_path }}"
+        state: directory
+      delegate_to: "localhost"
+      run_once: yes
+
+    - name: Grab host data
+      command: >
```

Comment:
I had a quick play with it at the time, it didn't work but I may have been doing it wrong. It was easier for me just to use rsync directly.